packages/scandipwa/src/component/DemoNotice/DemoNotice.style.scss

```diff
@@ -12,12 +12,23 @@
 :root {
     /* stylelint-disable-next-line length-zero-no-unit */
     --demo-notice-height: 0px;
+    --demo-notice-background-color: var(--primary-base-color);
+}
+
+html {
+    &.isDemoVisible {
+        body {
+            &::after {
+                background: linear-gradient(var(--demo-notice-background-color) 50%, var(--footer-background-color) 50%);
+            }
+        }
+    }
```

Comment:
When you make changes to the code, please, look at how similar things are already implemented.
For example, why did you choose the `isDemoVisible` class name? Is it already present in the project?
If yes, maybe these styles should go there? :)

Reply:
What you're saying is right.
My intention was to set `body::after` only if `isDemoVisible` class is on HTML tag, however in this case nothing else would be placed before body tag to assign `isDemoVisible` class to it, except HTML so it can be moved into `@at-root .isDemoVisible` block.